### DISCUSSION TITLE
flex-nowrap doesn't work

### DISCUSSION THREAD

**Discussion by kayotimoteo** [tailwindlabs/tailwindcss#7966] [Help] [answered]
How to make it work? Instead of shrinking the rest of the elements and creating a scrool bar?

https://play.tailwindcss.com/3U1X0wByIH

**Comment by kayotimoteo** [comment, marked as answer]
Resolved

**Reply by Nayan156** [reply]
How did you resolved it?

**Reply by wongjn** [reply]
Here's a possible solution: https://play.tailwindcss.com/CYBWTgTW5e
